zappa_settings.json.template

```diff
@@ -0,0 +1,16 @@
+{
+    "sandbox": {
+        "app_function": "main_app.app",
+        "memory_size": 512,
+        "keep_warm": false,
+        "aws_region": "eu-west-2",
+        "project_name": "da-ayr-flask-la",
```

Comment:
Is there a reason for the project name to end in "la"?

Reply:
If this is for lambda-API, this seems ok to me.

Reply:
it was what zappa created when I first ran zappa init. I think it shortened that from flask-lambda-app or something. So it is worth changing tbf.

Reply:
have updated name, undeployed project with previous name and deployed again with new one